src/funcCoreTools/validateFuncCoreToolsInstalled.ts

```diff
@@ -4,20 +4,24 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { callWithTelemetryAndErrorHandling, DialogResponses, IActionContext } from '@microsoft/vscode-azext-utils';
+import * as os from "os";
 import { MessageItem } from 'vscode';
 import { funcVersionSetting, PackageManager } from '../constants';
+import { ext } from '../extensionVariables';
 import { FuncVersion, tryParseFuncVersion } from '../FuncVersion';
 import { localize } from '../localize';
 import { cpUtils } from '../utils/cpUtils';
 import { openUrl } from '../utils/openUrl';
 import { getWorkspaceSetting } from '../vsCodeConfig/settings';
 import { getFuncCliPath, hasFuncCliSetting } from './getFuncCliPath';
 import { getFuncPackageManagers } from './getFuncPackageManagers';
+import { getLinuxDistroTag, LinuxDistroTag } from './getLinuxDistroTag';
 import { installFuncCoreTools } from './installFuncCoreTools';
 
 export async function validateFuncCoreToolsInstalled(context: IActionContext, message: string, workspacePath: string): Promise<boolean> {
     let input: MessageItem | undefined;
     let installed: boolean = false;
+    let failedInstall: string = localize('failedInstallFuncTools', 'Core Tools installation has failed and will need to be installed manually. Try re-running the last command with "sudo" or click "Learn more" for more options.');
```

Comment:
I think it could be confusing for the failure message to include
>Try re-running the last command with "sudo" or click "Learn more" for more options.

Could we just concatenate this at the end of the Linux failure scenario?

Reply:
We could but then I'll have to re-scope the variables above because they are not part of the same block.

So what exactly would you want the full verbiage for the failure message to look like?

Reply:
Could you do something like this on L52?
```
let linuxMessage = 'Try re-running the last command with "sudo" or click "Learn more" for more options.'
```

And then everything else would get appended as it currently does now.

Reply:
Example:

![image](https://user-images.githubusercontent.com/40250218/205177576-08e56874-b0b1-4fd5-8ec8-834fa2a0c494.png)

Reply:
Yeah, I think that would be fine. Can't remember if there is anyway we can add line breaks to the notification messages.

Reply:
Oh I see what you mean, will do.

Reply:
> Yeah, I think that would be fine. Can't remember if there is anyway we can add line breaks to the notification messages. Would be nice after `Note:`

I tried, but VS Code seems to strip away new lines

Reply:
Can we put in a button to copy that command to the clipboard?